### PR TITLE
feat: bring back the podspec

### PR DESCRIPTION
**Pull request by andrejborstnik**
Will need it for RN 64. Partially reverts & tweaks https://github.com/ExodusMovement/react-native-fast-crypto/commit/758c3a80e7e497aac331dce0014f8094ac323f36

### CODE REVIEW

File: RNFastCrypto.podspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "RNFastCrypto"
+  s.version      = package['version']
+  s.summary      = "RNFastCrypto"
+  s.description  = package['description']
+  s.homepage     = "https://github.com/ExodusMovement/react-native-fast-crypto.git"
```
Comment (Sekhmet) [inline RNFastCrypto.podspec]:
```suggestion
  s.homepage     = "https://github.com/ExodusMovement/react-native-fast-crypto"
```

File: RNFastCrypto.podspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  s.license      = "MIT"
+  s.author             = { "Paul Puey" => "paul@airbitz.co" }
+  s.platform     = :ios, "7.0"
+  s.source       = { :git => "https://github.com/ExodusMovement/react-native-fast-crypto.git", :tag => "v#{s.version}" }
+  s.source_files  = "RNFastCrypto/**/*.{h,m}"
```
Comment (Sekhmet) [inline RNFastCrypto.podspec:15]: If we need proper podspec we might want [this](https://guides.cocoapods.org/syntax/podspec.html#vendored_libraries) as well, but I don't think it matters.

```suggestion
  s.source_files  = "RNFastCrypto/**/*.{h,m}"
  s.vendored_libraries = 'RNFastCrypto/Libraries/libboost_regex.a', 'RNFastCrypto/Libraries/libboost_system.a', 'RNFastCrypto/Libraries/libboost_thread.a', 'RNFastCrypto/Libraries/libmymonerocorecpp.a', 'RNFastCrypto/Libraries/libnativecrypto.a', 
```

**Review by Sekhmet — COMMENTED** [review]
It used to be inside of `ios` folder. Does it matter?

**Comment by andrejborstnik** [comment]
> It used to be inside of `ios` folder. Does it matter?

Its easier if it is in root. It doesn't matter for pods, the code scans the dirs. Most RN libs have the podspec on root

**Review by Sekhmet — APPROVED** [review]

**Pull request merged**